BreadPlayer.Core/Common/PasswordStorage.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace BreadPlayer.Core.Common
+{
+    public class PasswordStorage
+    {
+        // These constants may be changed without breaking existing hashes.
+        private const int SALT_BYTES = 24;
+
+        /// <summary>
+        /// Returns a Tuple, item1 is the salt and item2 is the hash
+        /// </summary>
+        /// <param name="password">The password that needs to be hashed</param>
+        /// <returns>A Tuple, item1 is the salt and item2 is the hash</returns>
+        public static Tuple<string, string> CreateHash(string password)
```

Comment:
You should use direct Tuple instead of the predefined one. Something like this:
```csharp
public static (string Salt, string Hash) CreateHash(string password)
```
And further on, where you use `Item1`, use `Salt` and instead of `Item2` use `Hash`. It is more self-explanatory and easier to understand that way.